### ISSUE TITLE
Can I use an "Atomic Host" EC2 AMI to deploy Openshift Origin?

### ISSUE THREAD

**Issue by unisisdev** [openshift/openshift-ansible-contrib#168]
Hi, congratulations again for your work! I have a question please.

I have installed Openshift Origin on AWS using the reference implementation. 

On the ose-on-aws.py script I have provided a CentOS 7 image (ami-6d1c2007).

Can I use instead an Atomic Host image like ami-114f5c7b? ("CentOS Atomic Host 7 x86_64 HVM EBS 1603_01"

Thank you very much in advance!

Best Regards

**Comment by cooktheryan** [comment]
Yes when launching use the --containerized=true --ami=ami-114f5c7b --deployment-type=origin

**Comment by unisisdev** [comment]
@cooktheryan Hi Ryan, great, thank you very much for your info! It's very useful for me.

**Comment by unisisdev** [comment]
@cooktheryan Hi Ryan, thank you very much for your help, it was very really useful, thanks.

Just FYI (maybe somebody else could face this issue on the future), after configure the above parameters ("--containerized=true --ami=ami-114f5c7b --deployment-type=origin") and try to make the deployment I faced this issue:

```
TASK [prerequisite : Install the required rpms] ********************************
failed: [ose-master03.domain.com] (item=[u'NetworkManager']) => {
    "failed": true, 
    "item": [
        "NetworkManager"
    ]
}

MSG:

Could not find a module for unknown.
```

I fixed this issue but on a new attempt a message said that I must configure the "openshift_release" variable.

So, I applied the following changes to be able to launch Atomic Host successfully:

1) On aws-ansible/playbooks/node-setup.yaml I have changed "containerized: false" with:

```
containerized: {{ containerized }}
```

2) On aws-ansible/playbooks/openshift-setup.yaml I have added the following line: 

```
openshift_release: "v1.3.2"
```

3) On aws-ansible/playbooks/roles/prerequisite/tasks/main.yaml I have added the condition "when: containerized == "false"" here:

```
- name: Install the required rpms
  when: containerized == "false"
  package: name="{{ item }}" state=latest
  with_items:
  - NetworkManager
```
Maybe this info can be useful for somebody else.
Thank you very much again for your help!
